string equipment_name
int64 equipment_id
int64 equipment_type
geometry_msgs/Pose pose